sdk/maps/azure-maps-search/README.md

```diff
@@ -26,7 +26,7 @@ Various documentation is available to help you get started
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-maps-search</artifactId>
-    <version>1.0.0-beta.1</version>
+    <version>2.0.0-beta.1</version>
```

Comment:
Does this need to be a new major version as the previous major version never shipped GA? Could this be `1.0.0-beta.2`, or was the new major version a decision made elsewhere?

Reply:
@dubiety ?

Reply:
Yes, we had a thorough discussion in [JS SDK Teams channel](https://teams.microsoft.com/l/message/19:344f6b5b36ba414daa15473942c7477b@thread.skype/1684229932399?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1684229932399&teamName=Azure%20SDK&channelName=Language%20%E2%80%93%20JS%E2%80%89%EF%BC%86%E2%80%89TS%20%F0%9F%A5%B7&createdTime=1684229932399).
We want to sync the versioning between different languages, so we updated all the version to v2.0.0-beta.x